commands/project/dev/index.ts

```diff
@@ -70,13 +112,22 @@ export async function handler(
 }
 
 function projectDevBuilder(yargs: Argv): Argv<ProjectDevArgs> {
+  yargs.option('profile', {
+    type: 'string',
+    alias: 'p',
+    description: i18n(`commands.project.subcommands.dev.options.profile`),
+    hidden: true,
+  });
+
   yargs.example([
     [
       '$0 project dev',
       i18n(`commands.project.subcommands.dev.examples.default`),
     ],
   ]);
 
+  yargs.conflicts('profile', 'account');
```

Comment:
Have we added a test for `conflicts` here?

Reply:
Good call, done in [9494987](https://github.com/HubSpot/hubspot-cli/pull/1520/commits/94949878726ed9bab4a22eb036076b79a2817582)